.github/ISSUE_TEMPLATE/question.md

```diff
@@ -20,7 +20,7 @@ If you have a coding question related to React and React DOM, it might be better
 
 ### https://www.reactiflux.com/
 
```

Comment:
Not everyone might be comfortable asking on a chat like Discord. Can we point folks here instead and let them choose? https://reactjs.org/community/support.html#popular-discussion-forums